### DISCUSSION TITLE
"with-docker" example not running correctly

### DISCUSSION THREAD

**Discussion by donkee** [vercel/next.js#52907] [Help]
### Summary

I'm trying to reduce our docker image size and speed up our build process (basically not copying the entire node_modules folder to the image), but I'm having trouble when running the new image based on the "with-docker" example. I've even used that Dockerfile verbatim with the same results.

<img width="540" alt="image" src="https://github.com/vercel/next.js/assets/50036001/1646e006-5b43-4b0b-81c4-8ae0d2240d16">

It doesn't map to `localhost` and just gives a random string of characters as the domain - different every time. I've tried forcing the hostname to localhost, which DOES set it correctly, but it's still not accessible when I try going to the URL. Is this just a case of the docker example not being updated for Next 13 or am I doing something wrong?

### Additional information

```js
Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:20 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T6000
    Binaries:
      Node: 19.1.0
      npm: 9.2.0
      Yarn: 1.22.19
      pnpm: N/A
    Relevant Packages:
      next: 13.4.10
      eslint-config-next: 13.4.3
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 4.7.3
    Next.js Config:
      output: standalone
```
```


### Example

_No response_

**Comment by icyJoseph** [comment]
Share the modified docker file

**Reply by donkee** [reply]
It happens with the example given in the docs

https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile

and following [these directions](https://nextjs.org/docs/pages/building-your-application/deploying#docker-image)

**Reply by icyJoseph** [reply]
Mmm I'll give it a go, but I always use that file when people submit repositories with issues, and it has always worked.

**Reply by donkee** [reply]
Did you happen to try it and find success?
